### DISCUSSION TITLE
Miniscope DAQ

### DISCUSSION THREAD

**Discussion by LiliVig** [pyControl/code#113] [Q&A] [answered]
Hi, I am trying to use the Miniscope DAQ Software to get a Video from our MiniCAM, but I am stuck right at the beginning. On the window it is said that there are no available compression codes on my computer, can anybody help me fix that? I will provide two photos. I now it’s a basic question and probably hasn’t anything to do with the software itself, but I still hope to get an answer and would be really thankful if I did!
![image](https://github.com/pyControl/code/assets/156173788/cd02ecd7-448a-4901-be1d-5e27bbd1631b)

**Comment by alustig3** [comment, marked as answer]
Hi @LiliVig,

This forum is for pyControl, a different neuroscience software project. I think you are looking for https://github.com/Aharoni-Lab/Miniscope-DAQ-QT-Software. Their [wiki](https://github.com/Aharoni-Lab/Miniscope-DAQ-QT-Software/wiki) may be helpful, or you could try asking your question on their [issues page](https://github.com/Aharoni-Lab/Miniscope-DAQ-QT-Software/issues?q=is:issue+is:open+sort:updated-desc). Good luck!
